tests/ai_reports/factories.py

```diff
@@ -0,0 +1,16 @@
+import random
+
+import factory
+
+from adhocracy4.test import factories as a4_factories
+from apps.ai_reports.models import AiReport
+
+
+class AiReportFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = AiReport
+
+    category = factory.Faker("word")
+    explanation = factory.Faker("sentence", nb_words=6)
+    confidence = random.random()
+    comment = factory.SubFactory(a4_factories.ModuleFactory)
```

Comment:
I noticed there is a comment factory in tests/factories. Good to use this instead.